protocol/x/clob/keeper/process_operations.go

```diff
@@ -144,8 +144,15 @@ func (k Keeper) ProcessInternalOperations(
 
 		switch castedOperation := operation.Operation.(type) {
 		case *types.InternalOperation_Match:
+			affiliatesWhitelistMap, err := k.affiliatesKeeper.GetAffiliateWhitelistMap(ctx)
```

Comment:
Move this outside of the for loop. The intention is to get the whitelist map once for all operations